### PR TITLE
Feature/share popover

### PR DESCRIPTION
**Pull request by dontry**

### CODE REVIEW

File: src/components/SharePanel.jsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	}
+
+	render() {
+		const { link, isLoading } = this.state;
+
+
+		return (
+			<div className="share-panel">
+				<h3 style={{ marginTop: '4px' }}>Share the Diagram on Confluence<sup>*</sup></h3>
+				{isLoading ?
```
Comment (MrCoder) [inline src/components/SharePanel.jsx:37]: Loading 状态可不可以只影响Copy Link那个Button。现在这样影响整个Panel，给用户的感觉响应太慢。用户看到静态内容，然后Copy Link稍微等一下是可以的，因为用户的注意力还没有立刻转移到那里。

File: src/components/SharePanel.jsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+					</div>
+					: <>
+						<div>
+							<p>Paste the link on Confluence and select "Display as a Card"</p>
+							<img width={200} height={100} style="background: #acacac" />
+						</div>
+						<br />
+						<div>
+							<p>Preview</p>
+							<div className="preview" >
```
Comment (MrCoder) [inline src/components/SharePanel.jsx:48]: Preview的设计请参考这个页面上的那个Card： https://zenuml-demo.atlassian.net/wiki/spaces/DEMO/pages/511049729/Embed+ZenUML.cn

**Review by MrCoder — CHANGES_REQUESTED** [review]

**Comment by MrCoder** [comment]
@dontry Please resolve the conflict and update preview card according to https://github.com/ZenUml/web-app-laravel/pull/151

**Comment by MrCoder** [comment]
Once it is ready, please remove the "Draft" status.

**Comment by dontry** [comment]
> @dontry Please resolve the conflict and update preview card according to https://github.com/ZenUml/web-app-laravel/pull/151 

@MrCoder Can I remove all unecessary asyn/await from `app.tsx` file?

**Comment by MrCoder** [comment]
> @MrCoder Can I remove all unecessary asyn/await from `app.tsx` file?
Sure

**Review by MrCoder — APPROVED** [review]

**Pull request merged**